qa-pipelines/pipeline-presets/deploy.yml

```diff
@@ -1,5 +1,7 @@
 ---
 
+klog-collection-on-failure: true
```

Comment:
can you add this to terraform deploy/upgarde yaml instead? We need both klog and email-notifications there.